core/db/migrate/20210722090705_add_store_id_to_spree_customer_returns.rb

```diff
@@ -0,0 +1,11 @@
+class AddStoreIdToSpreeCustomerReturns < ActiveRecord::Migration[6.1]
+  def change
+    unless column_exists?(:spree_customer_returns, :store_id)
+      add_column :spree_customer_returns, :store_id, :bigint
+      add_index :spree_customer_returns, :store_id
+      add_index :spree_customer_returns, [:number, :store_id], unique: true
```

Comment:
@szymoniwacz isn't the nimber unique already ?

Reply:
@damianlegawiec You are right. Do we want it to be unique globally, or in scope of store?

Reply:
@szymoniwacz I think globally is safer